serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/evn python3
+
+liter_or_less = int(input("Enter the number of containers that holding 1 liter or less: "))
+more_than_liter = int(input("Enter the number of containers that holding more than 1 liter: "))
+lightweight_container_deposit = 0.10
```

Comment:
CSW: Constant value

Reply:
lightweight_container_deposit and heavy_container_deposit variables represent 0.10 and 0.25 numbers, as it was indicated to do in the previous comment.